website/content/v1.12/talos-guides/configuration/acquire.md

```diff
@@ -0,0 +1,88 @@
+
+Talos can also accept machine configuration documents directly via kernel arguments [`talos.config.early` and `talos.config.inline`]({{< relref "../../reference/kernel#talosconfigearly-and-talosconfiginline" >}}). This method is particularly useful for initial bootstrapping, e.g. specifying a custom set of [trusted CAs]({{< relref "./certificate-authorities" >}}).
+
+### Embedded Configuration
+
+Talos supports embedding the machine configuration directly into the bootable image (ISO, USB, `installer`).
+This is done using the [Imager tool]({{< relref "../install/boot-assets#example-adding-embedded-machine-configuration-with-imager" >}}).
+When the system boots, it reads the embedded configuration.
+
+### Maintenance Mode
+
+As the last resort, if the machine configuration is still incomplete for a full boot, Talos will drop into [maintenance mode]({{< relref "./insecure" >}}).
+
+## Order of Methods
+
+Talos checks for machine configuration in the following order:
+
+- `STATE` partition:
+  - if no configuration (or no `STATE` partition) is found, then go to the next step
+  - if a valid configuration is found, and it is incomplete, proceed to maintenance mode
+  - if a valid configuration is found, and it is complete, proceed to boot
+- embedded configuration:
+  - if no embedded configuration is found, then go to the next step
+  - if a valid configuration is found, and it is incomplete, proceed to maintenance mode
+  - if a valid configuration is found, and it is complete, proceed to boot
+- `talos.config.early` kernel argument:
+  - if no configuration is found, then go to the next step
+  - if a valid configuration is found, and it is incomplete, proceed to maintenance mode
+  - if a valid configuration is found, and it is complete, proceed to boot
+- platform configuration:
+  - if no configuration is found, then go to the next step
+  - if a valid configuration is found, and it is incomplete, proceed to maintenance mode
+  - if a valid configuration is found, and it is complete, proceed to boot
+- `talos.config.inline` kernel argument:
+  - if no configuration is found, then go to the next step
+  - if a valid configuration is found, and it is incomplete, proceed to maintenance mode
+  - if a valid configuration is found, and it is complete, proceed to boot
+- maintenance mode:
+  - wait for the user to provide a valid and complete configuration via `talosctl apply-config --insecure`
+  - once a valid and complete configuration is applied, proceed to boot
+
+## Examples
+
+### Bare-metal with DHCP
+
+If the machine is booted with Talos on bare-metal and the environment provides DHCP automatic network configuration, no initial configuration is needed.
+If Talos is already installed, it will boot using the configuration from the `STATE` partition.
+If Talos is not installed, it will attempt to acquire configuration from either from the platform (`talos.config`, pull method), or drop into maintenance mode (push method via API).
```

Comment:
```suggestion
If Talos is not installed, it will attempt to acquire configuration from either the platform (`talos.config`, pull method), or drop into maintenance mode (push method via API).
```

Reply:
clarified